fleet-server.yml

```diff
@@ -3,3 +3,12 @@ output:
     hosts: '${ELASTICSEARCH_HOSTS:elasticsearch:9200}'
     username: '${ELASTICSEARCH_USERNAME:}'
     password: '${ELASTICSEARCH_PASSWORD:}'
+
+fleet:
+  agent:
+    logging:
+      level: '${LOG_LEVEL:INFO}'
+
+features:
+  actions:
+    enabled: false
```

Comment:
We should work to remove this quickly. Good for now.

Reply:
I added this in due to the questions raised before on what happens if the existing agent connects to the fleet.
Also the scale testing "horde" harness needs to be changed to work correctly against these changes.
Can remove once the "horde" is ready.